### PR TITLE
Added Schizophrenia

### PR DESCRIPTION
**Pull request by Budcoderr**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -176,6 +176,19 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+#### [Schizophrenia](https://en.wikipedia.org/wiki/Schizophrenia)
+
+>A mental disorder characterized by delusions, hallucinations, or other symptoms of lack of contact with reality.
```
Comment (icyflame) [inline readme.md]: Space between angular bracket and sentence

Comment (icyflame) [reply readme.md]: Also, please shorten this sentence.

File: readme.md [hunk 2]
```diff
@@ -176,6 +176,19 @@ the anecdote, you can throw away the data."
...
+
+The term ‘schizophrenia’ was coined in 1910 by the Swiss psychiatrist Paul Eugen Bleuler, and is derived from the Greek words ‘schizo’ (split) and ‘phren’ (mind). The oldest available description of an illness closely resembling schizophrenia can be found in the Ebers papyrus, which dates back to the Egypt of 1550 BC. It is a serious disorder which causes a disconnect between what a person perceives as reality and what truly is reality. This mental gap can be extremely confusing and upsetting, causing intense psychological distress.
```
Comment (icyflame) [inline readme.md]: Too many single quotations. Use double quotes or drop them whenever possible (you should use italics for words in another languages, as in `_schizo_ (split) and _phren_ (mind)` which will render as

_schizo_ (split) and _phren_ (mind)

Comment (icyflame) [reply readme.md]: Wrap this at 70 characters.

File: readme.md [hunk 3]
```diff
@@ -176,6 +176,19 @@ the anecdote, you can throw away the data."
...
+
+[R: nihm.gov](https://www.nimh.nih.gov/health/topics/schizophrenia/index.shtml)
+
+[R: webmd.com](https://www.webmd.com/schizophrenia/mental-health-schizophrenia#1)
```
Comment (icyflame) [inline readme.md]: No webmd please.

**Review by icyflame — CHANGES_REQUESTED** [review]
Hey, thanks.

mha.net ref is solid. You can search for one more reference that is a little less clinical.

File: readme.md (hunk 3 above)
Comment (rudr4x5h) [reply readme.md]: @icyflame I wanted to know the reason behind this. Why no webmd !?

**Comment by Budcoderr** [comment]
Changes done. Please review

**Pull request closed without merging**